src/digitalmarketplace/components/footer/template.test.js

```diff
@@ -19,6 +19,8 @@ const urlForMock = jest.fn((param) => {
     value = 'http://www.amazon.co.uk'
   } else if (param === 'external.terms_and_conditions') {
     value = 'http://www.yahoo.co.uk'
+  } else if (param === 'external.accessibility_statement') {
+    value = 'http://www.github.com'
```

Comment:
Nit - should we be using http://example.com for this? It seems a bit odd to be putting github in our tests...

Reply:
Totally fair - was following the previous test cases, but I'll just change them all.